Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -516,30 +516,30 @@ if [[ $DO_AP_STM_ENV -eq 1 ]]; then
 exportline="export PATH=$OPT/$ARM_ROOT/bin:\$PATH";
 grep -Fxq "$exportline" ~/$SHELL_LOGIN 2>/dev/null || {
     if maybe_prompt_user "Add $OPT/$ARM_ROOT/bin to your PATH [N/y]?" ; then
-        echo $exportline >> ~/$SHELL_LOGIN
-        eval $exportline
+        echo "$exportline" >> ~/$SHELL_LOGIN
+        eval "$exportline"
     else
         echo "Skipping adding $OPT/$ARM_ROOT/bin to PATH."
     fi
 }
 fi
 
-exportline2="export PATH=$ARDUPILOT_ROOT/$ARDUPILOT_TOOLS:\$PATH";
+exportline2="export \"PATH=$ARDUPILOT_ROOT/$ARDUPILOT_TOOLS:\$PATH\"";
```

Comment:
Does this work correctly?

Shouldn't this be

```suggestion
exportline2="export PATH=\"$ARDUPILOT_ROOT/$ARDUPILOT_TOOLS:\$PATH\"";
```
?

Similarly on other lines in here

Reply:
Yes, i've tested it extensively. Linter throws a warning about backslashes, but there was no cleaner solution I could find.

Reply:
Updated with your suggestion